gopdf.go

```diff
@@ -841,6 +862,12 @@ func (gp *GoPdf) AddTTFFontByReaderWithOption(family string, rd io.Reader, optio
 		return err
 	}
 
+	return gp.addTTFFontByReaderWithOption(subsetFont, family, option)
+}
+
+// addTTFFontByReaderWithOption adds font file by reader with option.
+// The given SubsetFontObj is expected to be configured in advance.
+func (gp *GoPdf) addTTFFontByReaderWithOption(subsetFont *SubsetFontObj, family string, option TtfOption) error {
```

Comment:
Sorry not to catch this in the last review.. the method name has `Reader` but it no longer works for a reader:
```suggestion
func (gp *GoPdf) setSubsetFontOption(subsetFont *SubsetFontObj, family string, option TtfOption) error {
```

Reply:
Changed it to `setSubsetFontObject` instead of `setSubsetFontOption`.